core-test/src/test/scala/org/bitcoins/core/psbt/PSBTSerializerTest.scala

```diff
@@ -157,6 +159,24 @@ class PSBTSerializerTest extends BitcoinSUnitTest {
     assert(inputKey == PSBTInputKeyId.UnknownKeyId)
   }
 
+  it must "successfully serialize a PSBT with a ProofOfReservesCommitment" in {
+    val psbt = PSBT(validPsbts.head)
+
+    val inputElements = psbt.inputMaps.head.elements :+
+      ProofOfReservesCommitment(
+        hex"000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f")
+
+    val psbtWithPoRC =
+      PSBT(psbt.globalMap,
+           psbt.inputMaps.updated(0, InputPSBTMap(inputElements)),
+           psbt.outputMaps)
+
+    assert(
+      psbtWithPoRC.inputMaps.head
+        .getRecords(ProofOfReservesCommitmentKeyId)
+        .size == 1)
```

Comment:
nit: rather than asserting `size == 1` you should assert that `.headOption.contains(proofOfReservesCommitment)`